source/fundamentals/collations.txt

```diff
@@ -4,5 +4,299 @@ Collations
+or aggregation operation results by string values. A collation is a set of character
+ordering conventions that apply to a specific language and locale.
+
+Collations in MongoDB
+---------------------
+
+MongoDB sorts strings using *binary collation* by default. This collation
+method uses the `ASCII standard <https://en.wikipedia.org/wiki/ASCII>`_
+character values to compare and order strings. Certain languages and locales
+have specific character ordering conventions that differ from the ASCII
+standard.
+
+For example, in Canadian French, the right-most accented character determines
+the ordering for strings when the other characters are the same. Consider the
+following Canadian French words: 
+
+- cote
+- coté
+- côte
+- côté
+
+When using the default binary collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+
+   cote
+   coté
+   côte
+   côté
+
+When using the Canadian French collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+
+   cote
+   côte
+   coté
+   côté
+
+Specify a Collation
+-------------------
+
+To specify a collation, create a ``Collation`` object. You must define the ``Locale`` field
+of the ``Collation`` object; all other fields are optional. For example, the following code
+snippet specifies a ``Collation`` object with the ``"en_US"`` locale collation:
+
+.. code-block:: go
+
+   myCollation := &options.Collation{Locale: "en_US"}
+
+For a complete list of ``Collation`` object fields, visit the `Collation API documentation 
+<{+api+}/mongo/options#Collation>`__. To see all the supported locales and the
+default values for the ``Locale`` fields, visit :manual:`Supported Languages and Locales
+</reference/collation-locales-defaults/#supported-languages-and-locales>`.
+
+Set a Collation on a Collection or View
+---------------------------------------
+
+You can apply a collation when you create a new collection or view. This defines the default
+collation for any operations called on that collection or view. Set a collation through a 
+``CreateCollectionOptions`` or ``CreateViewOptions`` object. Then, call the
+``CreateCollection()`` or ``CreateView()`` method with your options object as an argument.
+
+.. _golang-create-collection:
+
+Create a Collection Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example creates a new collection called ``books`` and specifies a default
+collation with the ``"fr"`` locale. The ``Strength`` collation field has a value of ``1``
+to ignore differences in letter accents.
+
+.. code-block:: go
+
+   myCollation := &options.Collation{Locale: "fr", Strength: 1}
+   opts := options.CreateCollection().SetCollation(myCollation)
+   err := db.CreateCollection(context.TODO(), "books", opts)
+
+   if err != nil {
+      log.Fatal(err)
+   }
+
+Use the Default Collation Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you call an operation that supports collation on the ``books`` collection, the operation
```

Comment:
Issue: I believe the term "collation" is used in two different ways in this sentence. The first "supports collation" means "supports collecting and combining" where as the second "collation" means "a way of ordering characters".

Suggestion: change the first usage I.E
```suggestion
If you call an operation that uses a collation on the ``books`` collection, the operation
```

Reply:
Made this change